test/abi.encodeParameter.js

```diff
@@ -12,8 +12,7 @@ describe('lib/solidity/coder', function () {
                 assert.equal(coder.encodeParameter(t.type, t.value).replace('0x', ''), t.expected);
             });
         };
-
-
+        
```

Comment:
@savan13 There are only spaces changes in this PR, and no issue created about required changes.